src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVFastqUtils.java

```diff
@@ -26,10 +31,26 @@
         private final byte[] bases;
         private final byte[] quals;
 
+        public FastqRead( final GATKRead read ) {
+            this(Utils.nonNull(read).getName(),
+                 read.isReverseStrand() ? BaseUtils.simpleReverseComplement(read.getBases()) : read.getBases().clone(),
```

Comment:
clone is unnecessary: getBases makes a copy, as does getBaseQualities.  so you could simplify this constructor considerably, i think.  maybe something like:
```
public FastqRead( final GATKRead read ) {
    name = Utils.nonNull(read.getName());
    if ( !read.isReverseStrand() ) {
        bases = read.getBases();
        quals = read.getBaseQualities();
    } else {
        bases = BaseUtils.simpleReverseComplement(read.getBases());
        quals = read.getBaseQualities();
        SequenceUtils.reverseQualities(quals);
    }
}
```